lib/apparmortest.pm

```diff
@@ -72,12 +86,9 @@ sub aa_tmp_prof_prepare {
 
     if ($type == 0) {
         assert_script_run "mkdir $prof_dir_tmp";
-        assert_script_run "cp -r $prof_dir/{tunables,abstractions} $prof_dir_tmp/";
-        if (!(is_sle('<15-SP4') or is_leap('<16.0'))) {    # apparmor >= 3.0
-            assert_script_run "cp -r $prof_dir/abi $prof_dir/disable $prof_dir_tmp/";
-        }
-        if (is_sle('<15') or is_leap('<15.0')) {    # apparmor < 2.8.95
-            assert_script_run "cp -r $prof_dir/program-chunks $prof_dir/disable $prof_dir_tmp/";
+        foreach my $file ("tunables", "abstractions", "abi", "disable", "program-chunks" ) {
+            # if src_dir exists and dst_dir does not exists
+            check_copy_dir("$prof_dir/$file", "$prof_dir_tmp/$file");
```

Comment:
2nd $file can be deleted, correct?

Reply:
Nope, It needs to check the second `$file` exisis or not, 
I use `[ -d $src_dir_name ] && [ ! -d $dst_dir_name ]` to make sure that the source dir exists and dst dir does not exists.

Reply:
If the destnation dir exits, it won't copy.

Reply:
Then one more comment, your function mentioned you copy dir, but actually you copy the file, correct? if yes. please fix your description for your function then.

Reply:
No, actually I use this function to copy directories, but it also can copy files.

Reply:
Maybe I need to update the function name and description to `check_copy_dir_file` or other,
because this function can check and copy files and directories.

Reply:
Finished

> Maybe I need to update the function name and description to `check_copy_dir_file` or other, because this function can check and copy files and directories.

@rfan1, Finished, please check.